requirements.txt

```diff
@@ -3,3 +3,4 @@ requests~=2.24.0
 uwsgi~=2.0.18
 flask~=1.1.2
```

Comment:
Line number 4 in `requirements.txt` needs to be removed for `exrproxy` to build:
```
flask~=1.1.2
```